docs/reproin.rst

```diff
@@ -0,0 +1,139 @@
+
+The ``REPROIN`` directory is a simple reproin-compliant DICOM (.dcm) dataset without sessions. 
+(Derived dwi images (ADC, FA etc.) that the scanner produced have been removed.::
+
+    [user@local ~/reproin_dicom/REPROIN]$ tree -I "*.dcm"
+
+        REPROIN
+        ├── data
+        └── dicom
+            └── 001
+                └── Patterson_Coben\ -\ 1
+                    ├── Localizers_4
+                    ├── anatT1w_acqMPRAGE_6
+                    ├── dwi_dirAP_9
+                    ├── fmap_acq4mm_7
+                    ├── fmap_acq4mm_8
+                    ├── fmap_dirPA_15
+                    └── func_taskrest_16
+
+Convert and organize
+--------------------
+
+From the ``REPROIN`` directory::
+
+    heudiconv -f reproin --bids  -o data --files dicom/001 --minmeta
+
+* ``-f reproin`` specifies the converter file to use
+* ``-o data/`` specifies the output directory *data*.  If the output directory does not exist, it will be created.
+* ``--files dicom/001`` identifies the path to the DICOM files.
+*  ``--minmeta`` ensures that only the minimum necessary amount of data gets added to the JSON file when created.  On the off chance that there is a LOT of meta-information in the DICOM header, the JSON file will not get swamped by it. fmriprep and mriqc are very sensitive to this information overload and will crash, so minmeta provides a layer of protection against such corruption.
+
+
+Output Directory Structure
+--------------------------
+
+Heudiconv's Reproin converter produces a hierarchy of BIDS directories::
+
+    data
+    └── Patterson
+        └── Coben
+            ├── sourcedata
+            │   └── sub-001
+            │       ├── anat
+            │       ├── dwi
+            │       ├── fmap
+            │       └── func
+            └── sub-001
+                ├── anat
+                ├── dwi
+                ├── fmap
+                └── func
+
+**TODO --- WHAT IS A REGION AND EXAM???**
+
+* The dataset is nested under two levels in the output directory: *Region* (Patterson) and *Exam* (Coben). *Tree* is reserved for other purposes at the UA research scanner.
+* Although the Program *Patient* is not visible in the output hierarchy, it is important.  If you have separate sessions, then each session should have its own Program name.
+* **sourcedata** contains tarred gzipped (tgz) sets of DICOM images corresponding to each NIFTI image.
+* **sub-001** contains the BIDS dataset.
+* The hidden directory is generated: *REPROIN/data/Patterson/Coben/.heudiconv*.
+
+At the Scanner
+**************
+
+**TODO --- Is this generally useful or should be cut???**
+
+Here is this phantom dataset displayed in the scanner dot cockpit.  The directory structure is defined at the top: *Patterson >> Coben >> Patient*
+
+* *Region* = *Patterson*
+* *Exam* = *Coben*
+* *Program* = *Patient*
+
+
+
+Reproin Scanner File Names
+****************************
+
```

Comment:
@yarikoptic This section needs review from someone who is more familiar with bids/reproin/mri